packages/commonwealth/server/workers/commonwealthConsumer/commonwealthConsumer.ts

```diff
@@ -142,16 +142,29 @@ export async function setupCommonwealthConsumer(): Promise<void> {
 function startRolloverLoop() {
   log.info('Starting rollover loop');
 
+  const loop = async () => {
+    try {
+      await handleEvent(ContestWorker(), {
```

Comment:
seems like the same event (same triggering mechanisms). The naming convention I've used for these timer events is something like: ContestRolloverTimerTicked for clarity, and since you are manually handling it, no need for outbox

Reply:
That naming convention makes sense.

While they're both triggered by the same mechanism, they should still be sufficiently isolated. The contest check is low priority and only applies to an edge case, while the rollover is core functionality, so they're both functionally different.

Reply:
so you would trigger them with different frequencies?

Reply:
moved to single event